packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -27,7 +27,13 @@ const handleLinkControlOnKeyDown = ( event ) => {
 };
 
 const handleLinkControlOnKeyPress = ( event ) => {
+	const { keyCode } = event;
+
 	event.stopPropagation();
+
+	if ( keyCode === ENTER ) {
+
+	}
```

Comment:
Were these changes intended to have been included? Looks like dead code to me.

Reply:
Dead code. Being removed elsewhere.